supervisor/homeassistant/websocket.py

```diff
@@ -30,6 +31,9 @@ def __init__(
 
     async def async_send_command(self, message: Dict[str, Any]):
         """Send a websocket command."""
+        while self.client._waiting is not None:  # pylint: disable=protected-access
+            await asyncio.sleep(0.001)
```

Comment:
Is a bit hacky, let's use https://docs.python.org/3/library/asyncio-sync.html#lock